### ISSUE TITLE
Ubuntu/Debian Repository

### ISSUE THREAD

**Issue by havenerk** [yhat/rodeo#583]
I added the Rodeo repository as explained on the yhat web page.  It installed 2.4.10 a few months back when I did it.  But it didn't track version changes.  I just downloaded the 2.5.2 deb from the github page to update--I've been at 2.4.10 all this time.  Is the repo suppose to work or not?  I can always check the github page every once in awhile to see if there are new updates.

**Comment by coristig** [comment]
Yeah I'm behind and haven't pushed the update yet.  We'll have it up in the next couple days.  Sorry about the delay

**Comment by saurabhgayali** [comment]
It's been a year. an update?
I love this IDE but had to shift to UBUNTU and I need it badly.
